ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/utils/filesHandler.go

```diff
@@ -11,10 +11,15 @@ type FilesHandler struct {
 	sourceDir string
 	tempDir   string
 	outDir    string
+	isTestRun bool `default:"false"`
```

Comment:
same, fileHandler tests should be outside of fileHandler

Reply:
I have removed this per our discussion